components/Benefits/Benefits.tsx

```diff
@@ -9,18 +10,21 @@ type BenefitsProps = {
 };
 
 export const Benefits = ({ title, benefits }: BenefitsProps) => {
+    const [bowlSize, setBowlSize] = useState(208);
+    const screen = useScreen();
+
+    useEffect(() => {
+        if (screen.includes('medium')) setBowlSize(238);
+        if (screen.includes('large')) setBowlSize(285);
+    }, [screen]);
```

Comment:
While this is a cool solution, it's easier just to set the width in CSS using a media query breakpoint

Reply:
& more performant

Reply:
I've updated this thread to follow this convention